easybuild/easyconfigs/p/Panoply/Panoply-5.6.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'Panoply'
+version = '5.6.0'
+
+homepage = 'https://www.giss.nasa.gov/tools/panoply'
+description = "Panoply plots geo-referenced and other arrays from netCDF, HDF, GRIB, and other datasets."
+
+
+toolchain = SYSTEM
+
+source_urls = ['https://www.giss.nasa.gov/tools/panoply/download/']
+sources = ['%(name)sJ-%(version)s.tgz']
+checksums = ['0c4703f289750da18c7a3e7f4c61f3eceb63fd5d9b5e43fcbaa2bdf7615eaaa6']
+
+dependencies = [
+    ('Java', '11', '', SYSTEM),
+]
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'mv %(installdir)s/panoply.sh %(installdir)s/bin',
+    'sed -i "s,jars,../jars,g" %(installdir)s/bin/panoply.sh',
```

Comment:
@TopRichard Can you add a comment above this specific line to explain what it does/why it's needed?

The others are pretty obvious.